plugin/trino-singlestore/src/test/java/io/trino/plugin/singlestore/SingleStoreQueryRunner.java

```diff
@@ -54,7 +54,7 @@ public static DistributedQueryRunner createMemSqlQueryRunner(
 
             server.execute("CREATE SCHEMA tpch");
 
-            queryRunner.installPlugin(new MemSqlPlugin());
+            queryRunner.installPlugin(new SingleStorePlugin());
             queryRunner.createCatalog("memsql", "memsql", connectorProperties);
```

Comment:
Let's use new name here.
Old name is already covered by a test in TestSingleStorePlugin.